ui/lib/pki/addon/components/page/pki-certificate-details.hbs

```diff
@@ -32,9 +32,9 @@
 {{/if}}
 
 {{#each @model.formFields as |field|}}
-  {{#if field.options.masked}}
+  {{#if field.options.displayCard}}
     <InfoTableRow @label={{or field.options.label (capitalize (humanize (dasherize field.name)))}}>
-      <MaskedInput @value={{or (get @model field.name) null}} @displayOnly={{true}} @allowCopy={{true}} />
+      <CertificateCard @data={{or (get @model field.name) null}} />
```

Comment:
I don't think you would need the `or` here with the `null`. The value would end up being either `undefined` or `null` if the model property doesn't have a value.